doc/tutorials/cascading.md

```diff
@@ -1,53 +1,81 @@
-be used if the key hasn't been defined by the user.
+Looking at this order, we can see that if a configuration option is specified by the user (in the **user** namespace) as well as in the **system** namespace, then the key in the **user** namespace takes precedence over the one in the **system** namespace. If there is no such key in the **user** namespace the key in the **system** namespace acts as a fallback.
 
+But lets demonstrate this with an example:
+
+###### Add a key to the system namespace ######
+Configuration in the **system** namespace is readable for all users and the same for all users. Therefore this namespace provides a default or fallback configuration.
+
+In the default Elektra installation only an administrator can update configuration here:
 ```
+$ kdb get /sw/tutorial/cascading/#0/current/test
+Did not find key
+
+# Now add the key ...
 $ sudo kdb set system/sw/tutorial/cascading/#0/current/test "hello world"
 create a new key system/sw/tutorial/cascading/#0/current/test with string hello world
 
+# ... and verify that it exists
 $ kdb get /sw/tutorial/cascading/#0/current/test
 hello world
+```
+
+###### Add a key to the user namespace ######
+A user may now want to override the configuration in **system**, so he sets a key in the **user** namespace:
 
-$ kdb set user/sw/tutorial/cascading/#0/current/test "hello universe"
-Create a new key user/sw/tutorial/cascading/#0/current/test with string hello universe
+```
+$ kdb set user/sw/tutorial/cascading/#0/current/test "hello galaxy"
+Create a new key user/sw/tutorial/cascading/#0/current/test with string hello galaxy
 
+# This key masks the key in the system namespace
 $ kdb get /sw/tutorial/cascading/#0/current/test
-hello universe
+hello galaxy
 ```
+Note that configuration in the **user** namespace only affects _this_ user. Other users would still get the key from the **system** namespace.
 
-Furthermore, in the order `dir` is even higher than `user`, which means that
-configuration in the current folder can overwrite user configuration.
+###### Add a key to the dir namespace ######
+The **dir** namespace is associated with a directory. The configuration in the **dir** namespace applies to the associated directory and all its subdirectories.
+This is useful if you have project specific settings (e.g. your git configuration or a .htaccess file).
 
-`.configuration` in your current directory:
+As **dir** precedes the **user** namespace, configuration in **dir** can overwrite user configuration:
 
 ```
-test = hello dir
-```
+# create and change to a new directory ...
+$ mkdir kdbtutorial && cd $_
 
-Then run:
+# ... and create a key in this directories dir-namespace
+$ kdb set dir/sw/tutorial/cascading/#0/current/test "hello universe"
+Create a new key dir/sw/tutorial/cascading/#0/current/test with string hello universe
 
-```
-$ sudo kdb mount /.configuration dir/sw/tutorial/cascading/#0/current ini
```

Comment:
As this is the second tutorial a beginner reads, I prefer not to use mount.
This concept belongs to the dedicated tutorial.